src/constants.ts

```diff
@@ -62,6 +63,12 @@ enum RequestType {
   signup = 3,
 };
 
+enum LogType {
+  create = 1,
+  update = 2,
+  error = 3,
```

Comment:
prefer UPPERCASE

Reply:
I prefer lowercase letters. Is it a convention? Then I'll change